.codespellrc

```diff
@@ -0,0 +1,4 @@
+[codespell]
+skip = .git,*.pdf,*.svg
```

Comment:
will need to also skip test directory as well as *.xml, *.mediawiki, *.omn.

The *.json and *.tsv files also need to be skipped.

Reply:
tests code could also have typos not worth keeping IMHO. I will exclude generated `*.xml, *.mediawiki, *.omn` .

Similarly to tests,  for `json` and `tsv` I would recommend to not exclude them until you hit some cases where there are false positives -- you might actually get true positives you would like to fix in your metadata / examples/ tests.